.github/workflows/publish.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Install Dependencies
+        run: yarn --frozen-lockfile
+
+      - name: ovsx publish
+        uses: HaaLeo/publish-vscode-extension@v1
```

Comment:
Are there any benefits to use this action instead of the official `vsce` CLI?

Also: nice that this extension is also published to the [Open VSX registry](https://open-vsx.org/extension/unifiedjs/vscode-mdx).

Reply:
It keeps usage consistency between vsce and open-vsx, I supposed?